tests/integration/output/local_log/run_MotEvo_analysis.motif_HNRNPF_820.stdout.log

```diff
@@ -0,0 +1,53 @@
+letter A val is 0.250000
```

Comment:
Please remove ALL `stdout` and `stderr` files from the repository. These are logs and should not be kept under version control.  
It makes me wonder - why did you add these exact ones in the first place? These streams are related to `run_MotEvo_analysis` rule, which I already covered...